moped-toolbox/README.md

```diff
@@ -37,3 +38,17 @@ to backfill project milestones after implementing the milestone template feature
 ```shell
 $ python create_milestones.py -e local -d '2022-06-17 00:00:00'
 ```
+
+## purchase_order_backfill
+
+This tool populates the `moped_purchase_order table` [see issue #8259](https://github.com/cityofaustin/atd-data-tech/issues/8259) with existing contractor and purchase order information from the `moped_projects` columns purchase_order_number and contractor. This script is intended only to be used once, to move the data from the moped_projects table before we remove the purchase_order_number and contractor columns. 
+
+### Usage instructions
+
+1. Create a python 3.x environment with `requests` installed.
```

Comment:
I wanted to recognize that you do have all of the instructions one could need to get this running even though I mentioned a requirements.txt in my comment above. 🙏